eng/helix/content/runtests.cmd

```diff
@@ -3,9 +3,18 @@ REM Need delayed expansion !PATH! so parens in the path don't mess up the parens
 setlocal enabledelayedexpansion
 
 REM Use '$' as a variable name prefix to avoid MSBuild variable collisions with these variables
+set $target=%1
 set $sdkVersion=%2
 set $runtimeVersion=%3
+set $queue=%4
 set $arch=%5
+set $quarantined=%6
+set $ef=%7
+set $aspnetruntime=%8
+set $aspnetref=%9
+REM Batch only supports up to 9 arguments using the %# syntax, need to shift to get more
+shift
+set $helixTimeout=%9
```

Comment:
This is starting to get a bit out of control, should we consider a better way to flow things from our build to the helix test runner, idk what config file, environment variables?  Just seems like we are going to constantly keep adding arguments over time otherwise...  not something we have to do as part of this PR, but worth thinking about now maybe

Reply:
Agreed